example/jdbc/src/main/java/org/apache/iotdb/JDBCExample.java

```diff
@@ -30,19 +30,19 @@
 public class JDBCExample {
   public static void main(String[] args) throws ClassNotFoundException, SQLException {
     Class.forName("org.apache.iotdb.jdbc.IoTDBDriver");
-    try (Connection connection = DriverManager.getConnection("jdbc:iotdb://127.0.0.1:6667/", "root", "root");
+    try (Connection connection = DriverManager.getConnection("jdbc:iotdb://127.0.0.1:6667/?boolFormat=bool", "root", "root");
       Statement statement = connection.createStatement()) {
```

Comment:
If we don't indicate `?boolFormat=bool`, is there any problem? 
If not, I think we'd better keep the old example to show we don't have to do such setting.